tests/keras/layers/convolutional_test.py

```diff
@@ -138,8 +140,6 @@ def test_averagepooling_1d():
 
 
 @keras_test
-@pytest.mark.skipif((K.backend() == 'cntk'),
-                    reason="cntk only supports dilated conv on GPU")
```

Comment:
I moved the backend check from here to L179, as we should run other/non-dilated conv test code, they can work on CPU.